requirements/base.txt

```diff
@@ -33,3 +33,4 @@ sentry-sdk==0.7.9
 django-redis-cache==2.0.0
 redis==3.2.1
 html5lib==1.0.1
+mozilla-django-oidc==1.2.2
```

Comment:
Note that installing this [creates a hard dependency](https://github.com/mozilla/mozilla-django-oidc/blob/master/setup.py#L35) on `cryptography`, which we only have as a soft dependency/recommendation for Kolibri itself (our crypto operations have pure-Python fallbacks when `cryptography` is not available). As long as it gracefully degrades (e.g. the OpenID endpoints don't work, but everything else is fine), then it could be OK. But we likely don't want it to install `cryptography` naively (we handle it specially when building the WHL file, to bundle CEXT builds for different architectures).

CC @lyw07

Reply:
Yes I discussed that with José yesterday. My suggestion was to only install the pure python packages`mozilla-django-oidc` and `josepy` without dependencies and get the remaining dependencies from `kolibri/dist/cext` folder as we have the `cryptography` and its dependencies installed there. One downside is that we don't have cryptography for all platforms installed in kolibri (not Mac OSX). 

We think that we should change the distribution of the c packages in kolibri, which I would like to add as a discussion item in next week's dev meeting